lib/ansible/module_utils/facts/hardware/netbsd.py

```diff
@@ -116,12 +116,9 @@ def get_mount_facts(self):
         mount_facts = {}
 
         mount_facts['mounts'] = []
-        fstab = get_file_content('/etc/fstab')
-
-        if not fstab:
+        if not os.access("/proc/mounts", os.R_OK):
```

Comment:
Can you add a changelog entry for this change and test case?

Reply:
I thought I did provide a changelog entry. Where should one go, i.e. how do I provide one?

And can you point me to the current tests for the NetBSD facts? As I said: I am no Python programmer. I could try to tweak an already present test, but I cannot create one from scratch.

Reply:
proc/mounts is not guaranteed, so its should fallback to fstab and/or make the source configurable since fstab can be in different locations ... though really a parsable `mount` output would be the perfect solution

Reply:
> I thought I did provide a changelog entry. Where should one go, i.e. how do I provide one?

We document the changelog info at https://docs.ansible.com/ansible/latest/community/development_process.html#changelogs-how-to. It's simply a yaml file under [changelogs/fragments](https://github.com/ansible/ansible/tree/devel/changelogs/fragments) that provide info on what was changed/fixed.